packages/shared/src/graphql/users.ts

```diff
@@ -423,3 +434,46 @@ export const UNSUBSCRIBE_PERSONALIZED_DIGEST_MUTATION = gql`
     }
   }
 `;
+
+interface ReadingDay {
+  date: string;
+  reads: number;
+}
+
+export const getReadingStreak30Days = async (
+  id: string,
+): Promise<ReadingDay[]> => {
+  const today = new Date();
+  const last30Days = subDays(today, 30);
+  const res = await request(graphqlUrl, USER_STREAK_HISTORY, {
+    after: last30Days.toISOString(),
+    before: today,
```

Comment:
Since `after`  is given as an ISO string - so basically in UTC, I think we should do the same here
```suggestion
    before: today.toISOString(),
```

Reply:
Right. Totally missed that. Thanks! 🚀